target/arc/regs-impl.c

```diff
@@ -134,7 +134,18 @@ arc_general_regs_get(const struct arc_aux_reg_detail *aux_reg_detail,
         reg = env->bta_l2;
         break;
 
```

Comment:
Please put in comments which bits you're setting/hard coding.

Reply:
I did that in the respin (same branch) - please see if the styling is OK - so we are consistent in the codebase !